File: Exercises/Java/Java_How to Program_8th Edition/Eclipse/Chapter 3_Introduction to Classes and Objects/Self-Review Exercises/GradeBook5/src/GradeBook.java
public class GradeBook 
{
	private String courseName;
	private String instructorName;
	
	public GradeBook (String name, String nameInstructor)
	{
		courseName = name;
		instructorName = nameInstructor;
	}
	
	public void setCourseName (String name)
	{
		courseName = name;
	}
	
	public String getCourseName()
	{
		return courseName;
	}

	public void setInstructorName (String name)
	{
		instructorName = name;
	}
	
	public String getInstructorName()
	{
		return instructorName;
	}
	
	public void displayMessage()
	{
		System.out.printf("Welcome to the grade book for \n%s!\n", getCourseName());
		System.out.printf("This course is presented by: \n%s!\n", getInstructorName());
	}
}

File: Exercises/C++/Dietel & Dietel C++ How to Program/Chapter_3/Exercises/GradeBook/GradeBook/GradeBook.cpp
// This program contains the member function implementation for class GradeBook
#include <iostream>
#include "GradeBook.h"
using namespace std;

// Constructor for GradeBook
GradeBook::GradeBook( string nameOfCourse, string nameOfInstructor)
{
	setCourseName( nameOfCourse );
	setInstructorName ( nameOfInstructor );
}

// Set course name for GradeBook
void GradeBook::setCourseName( string nameOfCourse )
{
	courseName = nameOfCourse;
}

// Set instructor name for GradeBook
void GradeBook::setInstructorName ( string nameOfInstructor )
{
	instructorName = nameOfInstructor;
}

// Get course name for GradeBook
string GradeBook::getCourseName()
{
	return courseName;
}

// Get instructor name for GradeBook
string GradeBook::getInstructorName()
{
	return instructorName;
}

// Display course name and instructor name
void GradeBook::displayMessage()
{
	cout << "Welcome to the grade book for\n" << getCourseName()
		 << "\nThis course is presented by: " << getInstructorName() << "!" << endl;
}











